src/Helpers/FileOutput.php

```diff
@@ -33,13 +33,12 @@ public function __construct()
     /**
      * Outputs a file depending on its type.
      *
-     * @param  string  $path
-     * @param  array  $attributes
      * @return void
      */
-    public function write(string $path, array $attributes = []): void
+    public function write(CacheEntry $asset, bool $dev = false): void
```

Comment:
🚩 Why are we passing `dev` as an argument here? Isn't that a general setting, why have it as an argument?